docs/source/qvm.rst

```diff
@@ -87,6 +95,75 @@ and then execute it with the QVM directly from the command line:
     [2018-11-30 11:13:58] Classical memory (low -> high indexes):
     [2018-11-30 11:13:58]     ro:  1 0
 
+The QVM offers a simple benchmarking mode with ``qvm --verbose
+--benchmark``. Example output looks like this:
+
+.. code::
```

Comment:
@stylewarning, when I clone your pyQuil repo and build the html files, this code section doesn't actually show up for me.
<img width="691" alt="Screen Shot 2019-05-01 at 17 32 03" src="https://user-images.githubusercontent.com/44273795/57051655-59f8f600-6c37-11e9-906e-83dc1fb1be7a.png">

Reply:
sphinx requires an argument to `.. code::` (e.g. `..code:: text`) and I guess a blank line following